e2e_test/s3/fs_sink.py

```diff
@@ -199,12 +200,12 @@ def _table():
         test_timestamp,
         test_timestamptz
         from {_table()} WITH (
-        connector = 's3',
+        connector = 'snowflake',
```

Comment:
It is weird to test snowflake sink in fs sink test. Let's keep the previous codes instead.